shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -9,13 +9,17 @@
 static const char _kTextAffinityDownstream[] = "TextAffinity.downstream";
 static const char _kTextAffinityUpstream[] = "TextAffinity.upstream";
 
-static UIKeyboardType ToUIKeyboardType(NSString* inputType) {
+static UIKeyboardType ToUIKeyboardType(NSDictionary* type) {
```

Comment:
Optional: Maybe slightly cleaner to break out (NSString *inputType, BOOL signed)?